Automattic/wp-calypso
# Add mp4 Files to the Webpack Loader

#### Changes proposed in this Pull Request

This adds mp4 files to the Webpack loader (for examples of future usage, see #39617). 

#### Testing instructions

I changed the video in the Free Photo Library Card to test his. You can head to Customer Home, and start the video in that card. Verify through the link that the file is loaded, and that it plays properly.

<img width="754" alt="Screenshot 2020-02-26 at 16 44 46" src="https://user-images.githubusercontent.com/43215253/75367193-d4c28a00-58b7-11ea-9262-dcdb2edda575.png">
